### DISCUSSION TITLE
<gitluck>

### DISCUSSION THREAD

**Discussion by github-gitluck** [community/community#48764] [Packages]
### Select Topic Area

Product Feedback

### Body

https://github.com/github-gitluck?tab=packages

**Comment by ghostinhershell** [comment]
Hey there @github-gitluck, 

Thanks so much for posting in the Accessibility community. Did you have an Accessibility-related question or feedback to share that's related to [GitHub Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages)?

**Comment by cehfisher** [comment]
Hi @github-gitluck - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
